localstack/services/kinesis/kinesis_listener.py

```diff
@@ -150,7 +151,16 @@ def return_response(self, method, path, data, headers, response):
                 if sdk_v2:
                     record['ApproximateArrivalTimestamp'] = int(record['ApproximateArrivalTimestamp'])
                 if not isinstance(record['Data'], str):
-                    record['Data'] = bytearray(record['Data']['data']).decode('utf-8')
+                    # Remove double quotes from data written as bytes
+                    # https://github.com/localstack/localstack/issues/3588
+                    tmp = bytearray(record['Data']['data'])
+                    tmp = base64.b64encode(tmp).decode('utf-8')
```

Comment:
Is the `b64encode`->`b64decode` conversion roundtrip really required here? Could we simplify lines 156-159 to something like this: (haven't tested this, though)
```
tmp = record['Data']['data']
if len(tmp) >= 2 and tmp[0] == tmp[-1] == b'"':
    record['Data'] = tmp[1:-1]
```
Also, note that using `.strip('"')` can be problematic, e.g., if the transmitted content in the record looks something like `Hi, I'm "Alice"`